vcx/wrappers/java/src/main/java/com/evernym/sdk/vcx/issuer/IssuerApi.java

```diff
@@ -155,20 +155,20 @@ public void callback(int commandHandle, int err, int state) {
     }
     private static Callback issuerSendCredentialCB = new Callback() {
         @SuppressWarnings({"unused", "unchecked"})
-        public void callback(int commandHandle, int err, String credentialDefId) {
-            logger.debug("callback() called with: commandHandle = [" + commandHandle + "], err = [" + err + "], credentialDefId = [" + credentialDefId + "]");
-            CompletableFuture<String> future = (CompletableFuture<String>) removeFuture(commandHandle);
+        public void callback(int commandHandle, int err) {
```

Comment:
Note for other reviewers: this seems correct, see the callback definition here
https://github.com/hyperledger/indy-sdk/blob/master/vcx/libvcx/src/api/issuer_credential.rs#L196